modules/pruning-images.adoc

```diff
@@ -5,111 +5,152 @@
 [id="pruning-images_{context}"]
 = Pruning images
 
-In order to prune images that are no longer required by the system due to age,
-status, or exceed limits, administrators can run the following command:
+Administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits. There are two methods to manually prune images:
 
+* Running image pruning as a `Job` or `CronJob` on the cluster.
+* Running the `oc adm prune images [<options>]` command.
+
+.Prerequisites
+
+* To manually prune images, you must first log in to the CLI as a user with an access token. The user must also have the cluster role of `system:image-pruner` or greater (for example, `cluster-admin`).
+* Expose the image registry.
+
+.Procedure
+
+Running image pruning as a `Job` or `CronJob` on the cluster.
+
+* To manually prune images that are no longer required by the system due to age, status, or exceed limits create a YAML file for the `pruner` service account, for example:
++
+[source,yaml]
 ----
-$ oc adm prune images [<options>]
+kind: List
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: pruner
+    namespace: openshift-image-registry
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: openshift-image-registry-pruner
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: system:image-pruner
+  subjects:
+  - kind: ServiceAccount
+    name: pruner
+    namespace: openshift-image-registry
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: image-pruner
+    namespace: openshift-image-registry
+  spec:
+    schedule: "0 0 * * *"
+    concurrencyPolicy: Forbid
+    successfulJobsHistoryLimit: 1
+    failedJobsHistoryLimit: 3
+    jobTemplate:
+      spec:
+        template:
+          spec:
+            restartPolicy: OnFailure
+            containers:
+            - image: "quay.io/openshift/origin-cli:4.1"
+              resources:
+                requests:
+                  cpu: 1
+                  memory: 1Gi
+              terminationMessagePolicy: FallbackToLogsOnError
+              command:
+              - oc
+              args:
+              - adm
+              - prune
+              - images
+              - --certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+              - --keep-tag-revisions=5
+              - --keep-younger-than=96h
+              - --confirm=true
+              name: image-pruner
+            serviceAccountName: pruner
 ----
 
-Currently, to prune images you must first log in to the CLI as a user with an
-access token. The user must also have the cluster role `system:image-pruner` or
-greater (for example, `cluster-admin`).
+Running the `oc adm prune images [<options>]` command.
 
+* To manually prune images that are no longer required by the system due to age, status, or exceed limits by running the following command:
```

Comment:
Same here, remove _To manually prune images that are no longer required by the system due to age, status, or exceed limits by_? 
Just: _Run the following command_?